### DISCUSSION TITLE
Staff ->Studenti>Richieste

### DISCUSSION THREAD

**Discussion by Forrest-It** [iisgiua/giuaschool#432] [Richieste di aiuto]
Volevo chiedere se è possibile fare in modo che nella lista delle richieste [Ruolo staff ] Staff->Studenti>Richieste, siano riportate anche le richieste di uscita anticipata . Al momento nella lista vengono fuori solo le richieste di entrata anticicipata, quelle in cui nel modulo di richiesta il valore di "Codifica del tipo di richiesta" è uguale a E.

**Comment by trinko** [comment]
Le richieste di uscita anticipata che usano la codifica con codice "U" non vengono riportate nella lista perché sono gestite direttamente dai docenti tramite la pagina delle assenze o dallo staff tramite la pagina **staff->studenti->Ritardi e uscite**.
Il motivo è che normalmente l'alunno segnala di dover uscire al docente o direttamente allo staff, i quali controllano la presenza della richiesta e autorizzano l'uscita.

Se però si vuole fare in modo che vengano gestite solo dallo staff, tramite la pagina di gestione delle richieste, si può modificare l'impostazione del modulo, scegliendo come codifica un altro carattere a scelta (es. S).

**Comment by Forrest-It** [comment]
Grazie mille
